tests/aws/services/sqs/test_sqs.py

```diff
@@ -72,18 +75,23 @@ def sqs_snapshot_transformer(snapshot):
     snapshot.add_transformer(snapshot.transform.sqs_api())
 
 
+@pytest.fixture(params=["sqs", "sqs_json"])
+def aws_sqs_client(aws_client, request: str) -> "SQSClient":
+    yield getattr(aws_client, request.param)
```

Comment:
FYI: This fixture basically parameterizes every test which uses it to test both protocols, `query` and `json`. The change from the default `aws_client` fixture to this one in most of the tests adds 122 new test executions. This causes most of the changed lines in this test file.